enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/discovery/TopologyServiceMultiRetryStrategy.java

```diff
@@ -23,11 +23,22 @@
 
 import org.neo4j.causalclustering.identity.MemberId;
 import org.neo4j.helpers.AdvertisedSocketAddress;
+import org.neo4j.logging.LogProvider;
 
 public class TopologyServiceMultiRetryStrategy extends MultiRetryStrategy<MemberId,Optional<AdvertisedSocketAddress>> implements TopologyServiceRetryStrategy
 {
-    public TopologyServiceMultiRetryStrategy( long delayInMillis, long retries )
+    public TopologyServiceMultiRetryStrategy( long delayInMillis, long retries, LogProvider logProvider )
     {
-        super( delayInMillis, retries );
+        super( delayInMillis, retries, logProvider, millis ->
```

Comment:
Come to think of it, this may read a bit better if the lambda were a method reference, e.g. `super( delayInMillis, retries, logProvider, this::sleep )`